src/cache/cache_update.rs

```diff
@@ -16,7 +16,7 @@ use super::Cache;
 /// Creating a custom struct implementation to update the cache with:
 ///
 /// ```rust,ignore
-/// use serde_json::json;
+/// use crate::json::json;
 /// use serenity::{
 ///     cache::{Cache, CacheUpdate},
 ///     model::{
```

Comment:
`crate` should be `serenity`, and thus be a part of the import below. Usage of `serde_json` in the example further down should also be updated to be json-impl agnostic.  
```suggestion
/// use serenity::{
///     json::json,
///     cache::{Cache, CacheUpdate},
///     model::{
```